AddTransaction.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { describe, it, expect } from 'vitest'
+import { mount } from '@vue/test-utils'
+import AddTransaction from '@/components/AddTransaction.vue'
+
+describe('AddTransaction elements', () => {
```

Comment:
the describe should have the name of the file you wnat to test

```suggestion
describe('AddTransaction', () => {
```